src/EditorFeatures/Core/Implementation/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -51,12 +52,16 @@ internal void FindReferences(ITextSnapshot snapshot, int caretPosition)
                     var service = document.Project.LanguageServices.GetService<IFindReferencesService>();
                     if (service != null)
                     {
-                        if (!service.TryFindReferences(document, caretPosition, context))
+                        var globalOperationService = document.Project.Solution.Workspace.Services.GetService<IGlobalOperationNotificationService>();
+                        using (globalOperationService?.Start("FindReferences"))
```

Comment:
I'd leave it the way it is, Ravi. We don't usually localize such strings. As HeeJae says, they're useful for logging, but never displayed to the user.